websites/I/IDLIX/metadata.json

```diff
@@ -8,8 +8,13 @@
 	"description": {
 		"en": "Watch the latest movies, TV series, dramas, anime with high quality available in Indonesian subtitles and updated every day. Box Office Movies to the Best TV Series are all available on the IDLIX site."
 	},
-	"url": "tv.idlixofficial.co",
-	"version": "1.0.9",
+	"url": "idlix.com",
+	"regExp": "((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)[.](25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)[.](25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)[.](25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)[/])|((tv[.])?idlix(plus)?[.]net)|((tv[.])?idlix(official)?[.]co)",
+	"matches": [
+		"*://*.idlix*.*/*",
+		"*://*/*"
```

Comment:
Not anything else in the path you can use to match?

Reply:
nope